### DISCUSSION TITLE
Docker Hub: nextcloud/aio-talk

### DISCUSSION THREAD

**Discussion by ralfrupf1976** [nextcloud/all-in-one#4929] [Questions] [answered]
Nextcloud AIO has a limited number of user. What about the Containers on Docker Hub - can I use the Containers for talk and talk recording for my self-hosted cloud or even for a few clients? Are there any limits?

**Comment by szaimen** [comment, marked as answer]
Hi, just to leave this here, the limits can be removed by purchasing an enterprise license for AIO: https://nextcloud.com/all-in-one/
